test/unit/foreman/access_permissions_test_base.rb

```diff
@@ -0,0 +1,69 @@
+module AccessPermissionsTestBase
```

Comment:
The path should probably be test/unit/shared/access_permissions_test_base.rb to match other common bits of test code.